Formula/argocd-vault-plugin.rb

```diff
@@ -0,0 +1,34 @@
+class ArgocdVaultPlugin < Formula
+  desc "Argo CD plugin to retrieve secrets from Secret Management tools"
+  homepage "https://github.com/IBM/argocd-vault-plugin"
+  url "https://github.com/IBM/argocd-vault-plugin.git",
+    tag:      "v1.1.2",
+    revision: "b91d267dfcc69e8a0f029ca5afed585456f537a3"
```

Comment:
```suggestion
      tag:      "v1.1.2",
      revision: "b91d267dfcc69e8a0f029ca5afed585456f537a3"
```
Let's align the arguments to `url`.